modules/op-release-notes-1-1.adoc

```diff
@@ -17,6 +17,8 @@
 In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.1.
 
 === Pipelines
+
+* Workspaces can now be used instead of PipelineResources is now provided. It is recommended that you use Workspaces in OpenShift Pipelines, as PipelineResources are difficult to debug, limited in scope, and make Tasks less reusable. For more details on Workspaces, see Understanding OpenShift Pipelines.
```

Comment:
For the first sentence, I think you can remove "is now provided."

Reply:
Thank you for pointing that!! Not sure how I missed that!